### DISCUSSION TITLE
Onde saber mais sobre Restsharp no Balta.io

### DISCUSSION THREAD

**Discussion by omarceloribeiro** [balta-io/forum#484] [Tira Dúvidas] [answered]
Boa tarde, assisti a aula do CRUD no Blazor WebAssembly e lá é recomendando o uso do RestSharp. queria saber mais sobre as praticas do rest sharep no .net core mas não encontrei esse conteúdo no siste do Balta.
obs: sei que tem a documentação no site do restsharp, porém no video fala que é bastante usado nos cursos do balta. dai queria ver o conteudo dos cursos. obrigado.

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @omarceloribeiro, como vai?

Acredito que mostrei ele em algum evento, mas não me recordo se ficou gravado.
De qualquer forma vou trazer mais conteúdo sobre Blazor WASM em breve e falarei mais sobre ele.

**Comment by omarceloribeiro** [comment]
show de bola, muito obrigado. o que eu precisava saver era tipo boas praticas, boa pratica para instanciar o resetclient, para setar o base url, setar o relative path.

mais nesse sentido mesmo.
de qualquer forma, eu fiz uma tela aqui em blazor wasm, funcionou que é uma maravilha. bizarro como foi simples e limpo. fiquei muito impressionado. esse é o futuro 😆
tipo, pra landing page nao faz sentido, ok.(apesar que achei um blazor landing page muito top "blazorlandingpage . netlify . app" cria paginas apenas configurando arquivos arquivos json. top!
) mas pra app corporativo, app do tipo aplicacao mesmo, aquele tempinho de load é totalemnte factivel. inclusive apps como youtube, trello, possuem esse tipo de load, e sao em react ou angular.
pra app corporativo nao tem outro, blazor é campeao 😄

valeu!
________________________________
De: Andre Baltieri ***@***.***>
Enviado: domingo, 4 de fevereiro de 2024 17:36
Para: balta-io/forum ***@***.***>
Cc: Marcelo Ribeiro ***@***.***>; Mention ***@***.***>
Assunto: Re: [balta-io/forum] Onde saber mais sobre Restsharp no Balta.io (Discussion #484)


Boa tarde, @omarceloribeiro<https://github.com/omarceloribeiro>, como vai?

Acredito que mostrei ele em algum evento, mas não me recordo se ficou gravado.
De qualquer forma vou trazer mais conteúdo sobre Blazor WASM em breve e falarei mais sobre ele.

**Comment by andrebaltieri** [comment]
Bom dia, Marcelo, como vai?

Legal demais, vou tentar preparar algo nesta linha, beleza?
Ai te aviso aqui.

Bom saber que curtiu o Blazor, aqui ele está a todo vapor também! hahahahah

Abraços,

👉 Receba os conteúdos técnicos do balta.io direto em seu E-mail<https://go.balta.io/conteudo>

--
André Baltieri
Microsoft MVP - Visual Studio & Development Technologies

CEO - balta.io
***@***.***
________________________________
De: Marcelo Ribeiro ***@***.***>
Enviado: domingo, 4 de fevereiro de 2024 22:05
Para: balta-io/forum ***@***.***>
Cc: André Baltieri ***@***.***>; Comment ***@***.***>
Assunto: Re: [balta-io/forum] Onde saber mais sobre Restsharp no Balta.io (Discussion #484)


show de bola, muito obrigado. o que eu precisava saver era tipo boas praticas, boa pratica para instanciar o resetclient, para setar o base url, setar o relative path.

mais nesse sentido mesmo.
de qualquer forma, eu fiz uma tela aqui em blazor wasm, funcionou que é uma maravilha. bizarro como foi simples e limpo. fiquei muito impressionado. esse é o futuro 😆
tipo, pra landing page nao faz sentido, ok.(apesar que achei um blazor landing page muito top "blazorlandingpage . netlify . app" cria paginas apenas configurando arquivos arquivos json. top!
) mas pra app corporativo, app do tipo aplicacao mesmo, aquele tempinho de load é totalemnte factivel. inclusive apps como youtube, trello, possuem esse tipo de load, e sao em react ou angular.
pra app corporativo nao tem outro, blazor é campeao 😄

valeu!
________________________________
De: Andre Baltieri ***@***.***>
Enviado: domingo, 4 de fevereiro de 2024 17:36
Para: balta-io/forum ***@***.***>
Cc: Marcelo Ribeiro ***@***.***>; Mention ***@***.***>
Assunto: Re: [balta-io/forum] Onde saber mais sobre Restsharp no Balta.io (Discussion #484)


Boa tarde, @omarceloribeiro<https://github.com/omarceloribeiro>, como vai?

Acredito que mostrei ele em algum evento, mas não me recordo se ficou gravado.
De qualquer forma vou trazer mais conteúdo sobre Blazor WASM em breve e falarei mais sobre ele.
